app.js

```diff
@@ -318,18 +320,54 @@ function CookieStore(location, minCust, maxCust, avgCookie) {
   allStores.push(this);
 }
 
-CookieStore.prototype.randomNumber = function () {
-  return Math.floor(Math.random() * (this.maxCust - this.minCust) + this.minCust);
-};
-
-CookieStore.prototype.populateHourlySales = function () {
-  for (var i = 0; i < hours.length; i++) {
-    var customersPerHour = Math.round(this.randomNumber() * this.avgCookie);
-    this.hourlySales.push(customersPerHour);
+//render method
+Store.prototype.renderRow = function(){
+  var trElement = document.createElement('tr');
+  var tdElement = document.createElement('td');
+  tdElement.textContent = this.location;
+
+  trElement.appendChild(tdElement);
+  for(var i = 0; i < hours.length; i++){
+    tdElement.textContent = this.hourlySales[i];
+    trElement.appendChild(tdElement);
+    storeTable.appendChild(trElement);
+
+    tdElement = document.createElement('td');
+    tdElement.textContent = this.dailySales;
+    trElement.appendChild(tdElement);
   }
-};
+  Store.prototype.randomNumber = function () {
```

Comment:
You need one more end curly brace here to end your `renderRow` method. Your indentation should give you a hint that something is wrong here!